packages/cozy-stack-client/src/DocumentCollection.js

```diff
@@ -156,6 +164,80 @@ class DocumentCollection {
     }
   }
 
+  async fetchDocumentsWithMango(path, selector, options = {}) {
+    return this.stackClient.fetchJSON(
+      'POST',
+      path,
+      this.toMangoOptions(selector, options)
+    )
+  }
+
+  /**
+   * Handle index creation if it is missing.
+   *
+   * When an index is missing, we first check if there is one with a different
+   * name but the same definition. If yes, we copy it to re-use the same index
+   * and remove the old one.
+   * If there is none, we create the new index.
+   *
```

Comment:
Two lines is too much ;)

Reply:
Really? It seems to me it's sometimes useful to be a bit more verbose to clarify things